src/interface/meshblock_data.hpp

```diff
@@ -258,6 +261,23 @@ class MeshBlockData {
     return GetAllVariables(sparse_ids, flux).unique_ids();
   }
 
+  // Queries related to swarm data
+  std::shared_ptr<SwarmContainer> &GetSwarmData(int n = 0) {
```

Comment:
Why not just remove `n` from the argument list here?

Reply:
I require that both `MeshBlockData` and `MeshData` versions of this function have the same function signature and arguments.  In this way, we can template downstream functions on data type and interface with swarms correctly.  I did add a check on `n` in a recent commit.

Reply:
Ah ok for mesh data `n` is the block index? Makes sense. :+1: